src/Treasury.ts

```diff
@@ -35,4 +38,44 @@ export function handleSubscriptionPaid(event: FeesClaimed): void {
   movement.amount = event.params.jurorShare
   movement.createdAt = event.block.timestamp
   movement.save()
+  let subscriptions = Subscriptions.bind(event.address)
+  let periodDetails = subscriptions.getJurorShare(event.params.juror, event.params.periodId)
+  let token = periodDetails.value0
+  updateJurorToken(token, event.params.juror, event)
```

Comment:
If subscriptions go directly to the juror's wallet, is this necessary in this case?

Reply:
Good call! Removing it.